addons/website/views/snippets/s_form_aside.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_form_aside" name="Form Aside">
+    <section class="s_form_aside o_cc o_cc2 pt64 pb64">
+        <div class="container">
+            <div class="row">
+                <div class="col-12 col-lg-6 order-lg-0 pt-0 pb-0" style="order: 1;">
+                    <h2>Contact us</h2>
+                    <p class="lead">Chat with us through or fill in this form to get information about anything related to Odoo apps, services, our company.</p>
+                    <t t-snippet-call="website.s_website_form" string="Form"/>
+                </div>
+                <div class="col-12 col-lg-5 offset-lg-1 pt0 pb32 order-lg-0" style="order: 0;">
+                    <img src="/web/image/website.s_text_cover_default_image" class="img img-fluid rounded" alt=""/>
```

Comment:
Using a portrait image instead of the `s_text_cover` would probably look better.

I would suggest creating a new record, even if it fallsback to `s_text_cover_default_image`, so if one day we want to customize it in an existing or new theme we can tweak them individually without impacting other snippets images. Also it would allow you to choose better fitting portrait image regarding the theme 🤷